backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/emu/com/badlogic/gdx/graphics/Pixmap.java

```diff
@@ -180,7 +181,11 @@ public int getHeight () {
 		return height;
 	}
 
-	public Buffer getPixels () {
+	public ByteBuffer getPixels () {
+		if (pixels == null) pixels = context.getImageData(0, 0, width, height).getData();

```

Comment:
The backing UInt8ClampedArray is unfortunately read-only (didn't excpect that), so neither the buffer gets updated nor the pixmap gets updated, when the buffer changes.

It would be possible to put the content of the ByteBuffer back to the pixmap, but we would have to add a new method like `setPixels(Buffer buf)`.

I think I'll close this PR, I didn't expect that the backing buffer is read-only :(